fumadocs/app/(home)/page.tsx

```diff
@@ -0,0 +1,30 @@
+import { FadeIn } from "@/components/FadeIn";
+import { FAQ } from "./faq/FAQ";
+import { Community } from "./community/Community";
+import { Features } from "./features/Features";
+import { Hero } from "./hero/Hero";
+import { Letter } from "./letter/Letter";
+
+export default function Home() {
```

Comment:
why is this directory named (home)? Is this a route group? https://nextjs.org/docs/app/api-reference/file-conventions/route-groups ? Should it be? (as it's a single page, right?)